homeassistant/components/envisalink/__init__.py

```diff
@@ -67,12 +77,13 @@
                     cv.string, vol.In(["HONEYWELL", "DSC"])
                 ),
                 vol.Required(CONF_USERNAME): cv.string,
-                vol.Required(CONF_PASS): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
                 vol.Optional(CONF_CODE): cv.string,
                 vol.Optional(CONF_PANIC, default=DEFAULT_PANIC): cv.string,
                 vol.Optional(CONF_ZONES): {vol.Coerce(int): ZONE_SCHEMA},
                 vol.Optional(CONF_PARTITIONS): {vol.Coerce(int): PARTITION_SCHEMA},
-                vol.Optional(CONF_EVL_PORT, default=DEFAULT_PORT): cv.port,
+                vol.Optional(CONF_USERS): {vol.Coerce(int): USERS_SCHEMA},
+                vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
```

Comment:
We no longer allow integrations to add or change a platform YAML configuration.

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>